misk-policy/src/test/kotlin/misk/policy/opa/RealOpaPolicyEngineTest.kt

```diff
@@ -24,14 +24,22 @@ import wisp.moshi.defaultKotlinMoshi
 import jakarta.inject.Inject
 import jakarta.inject.Named
 import jakarta.inject.Singleton
+import okhttp3.ResponseBody.Companion.toResponseBody
 
 @MiskTest(startService = false)
 internal class RealOpaPolicyEngineTest {
   @MiskTestModule val module: Module = object : KAbstractModule() {
     override fun configure() {
+      install(FakeMetricsModule())
+      bind<OpaMetrics>().to<MiskOpaMetrics>()
       bind<OpaPolicyEngine>().to<RealOpaPolicyEngine>()
     }
 
+    @Provides @Singleton
+    fun opaConfig(): OpaConfig {
```

Comment:
supernit: you could use a more Kotlin-esque syntax here like `fun opaConfig(): OpaConfig = OpaConfig("fake", ...)`